contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -63,23 +64,33 @@ contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AutomationComp
     bool isActive;
   }
 
+  bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
+  bytes32 public constant EXECUTOR_ROLE = keccak256("EXECUTOR_ROLE");
+  uint96 private constant DEFAULT_TOP_UP_AMOUNT = 9000000000000000000;
+  uint96 private constant DEFAULT_MIN_BALANCE = 1000000000000000000;
```

Comment:
can we call these `DEFAULT_TOP_UP_AMOUNT_JULES` and `DEFAULT_MIN_BALANCE_JULES`?

Reply:
("jules" is like "wei" for LINK)